docker/artifacts/Dockerfile

```diff
@@ -1,8 +1,25 @@
 FROM ubuntu:20.04
 
-RUN apt-get update && apt-get install -y python3 python3-pip
-COPY constraints.txt constraints.txt
-ENV PIP_CONSTRAINT=/constraints.txt
+RUN apt-get update && apt-get install -y python3 python3-pip curl gzip
+
+# Install argo for cronjob submission
+# https://github.com/argoproj/argo-workflows/releases/tag/v3.3.1
+RUN curl -sLO https://github.com/argoproj/argo-workflows/releases/download/v3.3.1/argo-linux-amd64.gz && \
```

Comment:
Adding Argo into this image adds ~100mb.  I'd understand if a separate image would be warranted here for the addition of Argo and some microphysics specific folders.

Reply:
I think this makes some sense IMO to make `artifacts` the image that encapsulates our submission/workflow management environment. Do we really not have the argo CLI version pinned anywhere else...pretty surprising that we haven't had any workflows break bc of it, but I guess argo's a well maintained/stable project.